_skycap/the-basics/concepts-and-terminology.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+template: one-col
+title: Concepts and terminology
+categories: the-basics
+lead: "Understand the basic concepts and terms we use in this documentation"
+legacy: false
+tags: ["the basics","concepts", "terminology","explanation"]
+permalink: /:collection/:path
+---
+
+## Containers and containerization
+
+Containers, at their most basic, are pieces of software (or “code”) bundled together with the other components they require to function (such as system libraries and configuration files). This bundle (or “container”) is treated as a standalone executable file.
+
+There are many benefits to treating software applications this way. Containers are:
+
+* **Self-contained:** Each container runs as an independent process or thread. This greatly reduces the chance of conflict with other applications using the same infrastructure. 
```

Comment:
Generally, an em dash preferred typography here instead of a colon? http://cloud66.plainspeak.co.za/styleguide/

Reply:
Cool, have replaced all the colons in lists with emdashes.